Gruntfile.js

```diff
@@ -48,25 +52,40 @@ module.exports = function(grunt) {
           open: true,
         }
       }
+    },
+    concat: {
+        basic: {
+          src: ['src/js/DncApp.js'],
+          dest: 'dist/js/dnc.js'
+        },
+        extras: {
+          src: ['src/js/L.DNC.js', 'src/js/**/*.js'],
+          dest: 'dist/js/L.DNC.js'
```

Comment:
oh, looks like we forgot to remove it when we cleaned up everything. Now it's all Leaflet. Only this is needed now:

```bash
+        basic: {
+          src: ['src/js/L.DNC.js', 'src/js/**/*.js'],
+          dest: 'dist/js/L.DNC.js'
+        },
```
let me fix that quick like